### PR TITLE
Adjust build resources

### PR DESCRIPTION
**Pull request by michal-hudy**
**Description**

Changes proposed in this pull request:

- Adjusted default build resources (function is built in less than 40s)
- Adjusted build resources for minikube (function is built in  ~2min)
- Disabled Knative Serving tests - it is tested by `function-controller` tests

**Related issue(s)**

#7579

### CODE REVIEW

File: resources/function-controller/templates/tests/test.yaml [hunk 1]
```diff
@@ -16,12 +16,6 @@ spec:
       annotations:
         sidecar.istio.io/inject: "true"
     spec:
-      {{ if .Values.global.isLocalEnv }}
```
Comment (magicmatatjahu) [inline resources/function-controller/templates/tests/test.yaml:19]: This is not needed for minikube?

File: resources/function-controller/values.yaml [hunk 2]
```diff
@@ -66,15 +63,41 @@ containers:
...
+      imagePullAccountName:
+        value: "{{ template \"fullname\" . }}"
+      functionRequeueDuration:
+        value: 1m
+      functionBuildExecutorImage:
+        value: gcr.io/kaniko-project/executor:v0.17.1
+      functionBuildCredsInitImage:
+        value: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/creds-init:v0.10.1
+      functionDockerAddress:
+        # TODO: I don't know why {{ .Values.global.dockerServicePort }} doesn't work. It is rendered as {{\n  .Values.global.dockerServicePort }}
```
Comment (magicmatatjahu) [inline resources/function-controller/values.yaml:96]: Probably this is caused by `global` context in templating.

File: resources/function-controller/templates/tests/test.yaml (hunk 1 above)
Comment (michal-hudy) [reply resources/function-controller/templates/tests/test.yaml:19]: Currently, we don't know how to handle minikube and it points to storage that is completely not used in this test... so I removed this to not confuse others.

File: resources/function-controller/templates/deployment.yaml [hunk 3]
```diff
@@ -82,21 +82,21 @@ spec:
...
+            {{ include "createEnv" ( dict "name" "APP_FUNCTION_BUILD_RUNTIME_CONFIG_MAP_NAME" "value" .Values.containers.manager.envs.runtimeConfigMapName "context" . ) | nindent 12 }}
+            {{ include "createEnv" ( dict "name" "APP_FUNCTION_BUILD_REQUESTS_CPU" "value" .Values.containers.manager.envs.buildRequestsCPU "context" . ) | nindent 12 }}
+            {{ include "createEnv" ( dict "name" "APP_FUNCTION_BUILD_REQUESTS_MEMORY" "value" .Values.containers.manager.envs.buildRequestsMemory "context" . ) | nindent 12 }}
+            {{ include "createEnv" ( dict "name" "APP_FUNCTION_BUILD_LIMITS_CPU" "value" .Values.containers.manager.envs.buildLimitsCPU "context" . ) | nindent 12 }}
+            {{ include "createEnv" ( dict "name" "APP_FUNCTION_BUILD_LIMITS_MEMORY" "value" .Values.containers.manager.envs.buildLimitsMemory "context" . ) | nindent 12 }}
+            {{ include "createEnv" ( dict "name" "APP_CONFIG_WATCHER_NAMESPACE_REQUEUE_DURATION" "value" .Values.containers.manager.envs.namespaceRequeueDuration "context" . ) | nindent 12 }}
```
Comment (magicmatatjahu) [inline resources/function-controller/templates/deployment.yaml:92]: Where is env for excluded namespaces? :)

File: resources/function-controller/templates/tests/test.yaml (hunk 1 above)
Comment (aerfio) [reply resources/function-controller/templates/tests/test.yaml:19]: but it "works" if you add that specific lambda name to /etc/hosts

maybe we could keep it for posterity?

Comment (michal-hudy) [reply resources/function-controller/templates/tests/test.yaml:19]: hm... I will check as I'm waiting for rename merge ;)

Comment (michal-hudy) [reply resources/function-controller/templates/tests/test.yaml:19]: @aerfio ok, I'm still thinking that it doesn't make sense to keep it, the function has generated FQDN (like `test-function-1234566.kyma.local`), not static.

**Pull request merged**